src/yival/enhancers/utils.py

```diff
@@ -63,5 +74,5 @@ def construct_output_format(variations: List[str]) -> str:
 
     prompt = "Do not write code. Please response with given format:\n"
     for var in variations:
-        prompt += f"{var}= \n"
-    return prompt
+        prompt += (var + '=' + '{' + "your generated " + f"{var}" + '}')
```

Comment:
Is hardcoding "your generated" necessary? 
var1={your generated var1}
var2={your generated var2}

If not, can we try this:
def construct_output_format(variations: List[str]) -> str:
    prompt_lines = [f"{var}= " for var in variations]
    prompt = "Do not write code. Please response with given format:\n" + '\n'.join(prompt_lines)
    return prompt

Reply:
Prompt1:
```
Now you will help me generate a prompt which is used to generate a corresponding TikTok video title according to the TikTok content topic [tiktok_cotnent_topic] and its targeted audience [ticktok_audience]. The video will also include a short description [short_description]. I already have some prompt and its evaluation results.
Input:
task=Complete the following task:
generate tiktok video title
tiktok_cotnent_topic: {tiktok_cotnent_topic}
ticktok_audience: {ticktok_audience}
short_description: {short_description}
Evaluation:
accuracy: 0.0 targeting: 0.6000000000000001 catchiness: 0.6000000000000001 searchability: 0.4 compliance: 0.8

Give me a new prompt that is different from all pairs above, and has a evaluation value higher than any of above.
Do not write code. Please response with given format:
task=
```

GPT4 Response:
```
Complete the following task:
craft a catchy TikTok video title
content_theme: {tiktok_cotnent_topic}
intended_audience: {ticktok_audience}
brief_summary: {short_description}
```

Prompt2:
```
Now you will help me generate a prompt which is used to generate a corresponding TikTok video title according to the TikTok content topic [tiktok_cotnent_topic] and its targeted audience [ticktok_audience]. The video will also include a short description [short_description]. I already have some prompt and its evaluation results.
Input:
task=Complete the following task:
generate tiktok video title
tiktok_cotnent_topic: {tiktok_cotnent_topic}
ticktok_audience: {ticktok_audience}
short_description: {short_description}
Evaluation:
accuracy: 0.0 targeting: 0.6000000000000001 catchiness: 0.6000000000000001 searchability: 0.4 compliance: 0.8

Give me a new prompt that is different from all pairs above, and has a evaluation value higher than any of above.
Do not write code. Please response with given format:
task= {your prompt}
```

GPT4 Response:
```
task= Please perform the task below:
Create a catchy TikTok video title
Content theme: {tiktok_cotnent_topic}
Intended audience: {ticktok_audience}
Brief summary: {short_description}
```

I found that use `your prompt` will be more clear for GPT-4 to follow the instructions